lib/matplotlib/colors.py

```diff
@@ -1259,7 +1266,7 @@ def reversed(self, name=None):
             name = self.name + "_r"
 
         colors_r = list(reversed(self.colors))
-        new_cmap = ListedColormap(colors_r, name=name, N=self.N)
+        new_cmap = ListedColormap(colors_r, name=name)
```

Comment:
Am I confused or was passing _N_ here (and below) already redundant?

Reply:
You are not confused. 😄  This was redundant. `self.colors` is of length `self,N`. Actually, this pattern is part of my motivation to deprecate `N`. You never know, whether `N` was passed "just because it's there" or with the intention to adjust the length of the given color list.